tests/cypress/e2e/llamalend/llamalend-markets.cy.ts

```diff
@@ -119,6 +119,7 @@ describe(`LlamaLend Markets`, () => {
 
   it('should find markets by text', () => {
     cy.get('[data-testid="btn-expand-search-Llamalend Markets"]').click({ waitForAnimations: true })
+    cy.get("[data-testid^='table-text-search-']").should('be.focused') // element is focused when animation completes
```

Comment:
The input is focused but the div isn't
```suggestion
    cy.get("[data-testid^='table-text-search-'] input").should('be.focused') // element is focused when animation completes
```